app/models/settings/mandatory.rb

```diff
@@ -0,0 +1,28 @@
+module Settings
+  # We use this model to back the "Get Started" form of the config admin page.
+  class Mandatory
+    include ActiveModel::Naming
+
+    MAPPINGS = {
+      community_name: Settings::Community,
+      community_description: Settings::Community,
+
+      suggested_tags: SiteConfig,
+      suggested_users: SiteConfig
+    }.freeze
+
+    MAPPINGS.each do |setting, settings_model|
+      delegate setting, "#{setting}=", to: settings_model
+    end
+
+    def self.keys
```

Comment:
This was primarily added to stay consistent with the settings models backed by the gem.